src/routers/__tests__/TabRouter-test.js

```diff
@@ -141,6 +141,42 @@ describe('TabRouter', () => {
     });
   });
 
+  test('Handles passing params to nested tabs', () => {
+    const ChildTabNavigator = () => <div />;
+    ChildTabNavigator.router = TabRouter({ Foo: BareLeafRouteConfig, Bar: BareLeafRouteConfig });
+    const router = TabRouter({ Foo: BareLeafRouteConfig, Baz: { screen: ChildTabNavigator } });
+    const action = router.getActionForPathAndParams('Baz', { foo: '42' });
+    const navAction = { type: NavigationActions.NAVIGATE, routeName: 'Baz', params: { foo: '42' } };
+    expect(action).toEqual(navAction);
```

Comment:
I'm having trouble getting this to pass (`action` doesn't contain the params field) - any suggestions?